pocket_coffea/utils/plot_utils.py

```diff
@@ -513,9 +514,10 @@ def _get_stacks(self, cat, spliteras=False):
                 self.nevents = dict(
                     sorted(self.nevents.items(), key=lambda x: x[1], reverse=reverse)
                 )
-                color = iter(cm.gist_rainbow(np.linspace(0, 1, len(self.nevents.keys()))))
-                # Assign random colors to each sample
-                self.colors = [next(color) for d in self.nevents.keys()]
+                # create cycler from the colormap and instantiate it to get iterator
+                color = cycler("color", hep.styles.cms.cmap_petroff)()
+                # Assign colors from cycle to samples
```

Comment:
I assume that if no colors are explicitely assigned, we want to take them from the cycler. But is this keeping the same mapping with the samples for all the plots?

Reply:
its not cycling across different shape objects, if thats what you mean
each time the cycler is instantiated, its a new cycler starting from the first element in the list which is taken from `hep.styles.cms.cmap_petroff` 
so the first sample in the sorted dict `self.nevents` is always assigned the first color in the `cmap_petroff` list

Reply:
ok perfect, the colours will be consistent between plots.